src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -64,6 +64,14 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 		valueToDisplay = escape(valueToDisplay.length > 250 ? valueToDisplay.slice(0, 250) + '...' : valueToDisplay);
 		titleValue = valueToDisplay;
 	}
+	if (columnDef?.iconCssCollection?.length && columnDef?.iconCssIndexField) {
+		// we have icon to display
+		let idx = dataContext[columnDef.iconCssIndexField];
+		if (idx > columnDef.iconCssCollection.length || idx < 0) {
```

Comment:
Can you check of the code is right when idx = columnDef.iconCssCollection.length? Also when idx is out of bounds, then it seems like an error condition and we should perhaps be throwing an error.

Reply:
this code is not used anymore. removing.